management/status_checks.py

```diff
@@ -646,6 +695,27 @@ def check_web_domain(domain, rounded_time, ssl_certificates, env, output):
 	# website for also needs a signed certificate.
 	check_ssl_cert(domain, rounded_time, ssl_certificates, env, output)
 
+def check_deliverability_domain(domain, deliverable, env, output):
+	result = True  # returns True if everything is set up OK
+	action = 'allow' if deliverable else 'prevent'
+
+	# Ensure the SPF record for this domain either allows or prevents email
+	expected = "\"v=spf1 %s-all\"" % ('mx ' if deliverable else '')
+	values = query_dns(domain, "TXT").split('; ')
```

Comment:
Let's modify query_dns to return the answers as a list rather than having to split them.